build.gradle

```diff
@@ -17,7 +17,7 @@ allprojects {
     apply plugin: 'groovy'
     apply plugin: 'idea'
     group = 'com.bmuschko'
-    version = '2.2.5'
+    version = '2.2.6'
```

Comment:
I think the functionality calls for a minor version upgrade rather than a patch upgrade.